internal/pkg/template/templates/workloads/services/backend/cf.yml

```diff
@@ -74,7 +97,20 @@ Resources:
             Action:
               - sts:AssumeRole
       Path: /
+      ManagedPolicyArns:
+        - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
       Policies:
+      {{- if .ALBEnabled}}
+        - PolicyName: "RulePriorityGeneratorAccess"
```

Comment:
The `HTTPRulePriorityAction` that's included for backend services I think have the following `ListenerARN`:

https://github.com/aws/copilot-cli/blob/9a479c48371e1f1ae339270219c4c1eb49dbe43d/internal/pkg/template/templates/workloads/partials/cf/http-listener.yml#L7
and

https://github.com/aws/copilot-cli/blob/9a479c48371e1f1ae339270219c4c1eb49dbe43d/internal/pkg/template/templates/workloads/partials/cf/http-listener.yml#L36

We'll want to update these listener ARNs to be for the InternalALB on top of Janice's changes

Reply:
resolved in https://github.com/aws/copilot-cli/pull/3608